pkg/task/v1alpha1/runtask_command.go

```diff
@@ -19,20 +19,27 @@ package v1alpha1
 import (
 	"errors"
 	"fmt"
-	. "github.com/openebs/maya/pkg/msg/v1alpha1"
 	"time"
+
+	msg "github.com/openebs/maya/pkg/msg/v1alpha1"
 )
 
 // RunCommandAction determines the kind of action that gets executed by run task
 // command
 type RunCommandAction string
 
 const (
+	// DeleteCommandAction represents delete command action
```

Comment:
`DeleteCommandAction represents a run command as a delete action`

Can you please rephrase the comments for all the actions?